src/v2/manifest/manifest_schema1.rs

```diff
@@ -49,4 +50,27 @@ impl ManifestSchema1Signed {
             .map(|l| l.blob_sum.clone())
             .collect()
     }
+
+    /// Get a collection of all image labels stored in the history array of this manifest.
+    ///
+    /// Note that for this manifest type any `layer` beyond 0 probably returns None.
+    pub fn get_labels(&self, layer: usize) -> Option<(HashMap<String, String>)> {
```

Comment:
I don't think we need to expose that information in the method name. Otherwise `get_layers` could be renamed to `get_reversed_fs_blob_sums`, and then where do we draw the line?

Reply:
Fair point, let's keep it as is then.

Reply:
I'm even tempted to remove the `get_` from the method names. What's your opinion?

Reply:
I'll follow up with this in a separate PR as it touches on code that doesn't belong in this PR.